.github/workflows/playwright-ci.yml

```diff
@@ -17,8 +17,26 @@ jobs:
           node-version: "14"
       - uses: microsoft/playwright-github-action@v1
 
-      - name: Install dependencies
-        run: yarn global add http-server && yarn && yarn bootstrap && yarn build
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install yarn dependencies
+        run: yarn global add http-server && yarn
+
+      - name: Install Lerna dependencies
```

Comment:
Can you rename that step please?
Something like : `Link locale packages and install their dependencies.`

Reply:
Sure, no worries. Just to clarify, which step you would like to be renamed? Since steps need to stay separate, I could name them something like:

`Link locale yarn packages and install their dependencies`
and
`Link locale Lerna packages and install their dependencies`

Reply:
I was talking about the `yarn bootstrap` command. IMO we can just rename the line 35.